tsl/src/hypercore/utils.c

```diff
@@ -12,21 +12,23 @@
 #include <catalog/pg_class.h>
 #include <commands/defrem.h>
 #include <nodes/makefuncs.h>
+#include <storage/lockdefs.h>
 #include <utils/builtins.h>
 #include <utils/lsyscache.h>
 #include <utils/syscache.h>
 
+#include "chunk.h"
 #include "extension_constants.h"
+#include "src/utils.h"
 #include "utils.h"
-#include <src/utils.h>
 
 /*
  * Make a relation use hypercore without rewriting any data, simply by
  * updating the AM in pg_class. This only works if the relation is already
  * using (non-hypercore) compression.
  */
 void
-hypercore_set_am(const RangeVar *rv)
+hypercore_convert_rv(const RangeVar *rv)
```

Comment:
I think this renaming is a bit misleading because no real conversion is done,  i.e., no data is rewritten, unlike the other places named "convert". This is purely a metadata change which is why it was named "set".

Reply:
Since it now does more than just setting the access method, the original name is also misleading. I'm open to alternatives, if you have one.